scripts/test

```diff
@@ -77,27 +123,37 @@ while ! curl -sf http://localhost:8080/ping; do
     sleep 2
 done
 
-while ! kubectl --kubeconfig /etc/rancher/k3s/k3s.yaml rollout status -w -n cattle-system deploy/rancher-webhook; do
+# Echo a newline to handle the "pong" from the above ping
+echo ""
+
+rwWC=0
```

Comment:
Can we give this a more descriptive name? It doesn't have to be verbose (`i` would be fine, or `attempt`, etc.), it's just not obvious without looking at the associated if statements. It might also be a good idea to make the logic generic, like so:
```
maxWaitSeconds=300 # 5 minutes
sleepSeconds=2
i=0
echo "Waiting up to 5 minutes for rancher-webhook deployment"
while ! kubectl --kubeconfig /etc/rancher/k3s/k3s.yaml rollout status -w -n cattle-system deploy/rancher-webhook &>/dev/null; do
  if [[ "$i" == $((maxWaitSeconds/sleepSeconds)) ]]; then
    echo "rancher-webhook was not available within $((maxWaitSeconds/60)) minutes. failing fast"
    exit 1
  fi
  if [[ $((i/(60/sleepSeconds))) == 0 ]] && [[ "$i" != "0" ]]; then
    echo "rancher-webhook was not ready within $((i/(60/sleepSeconds))) minutes"
  fi
    sleep $sleepSeconds
    i=$((i + 1))
done
```
Just a thought, mostly because changing in the future would require some back of the napkin math I wouldn't want to do lol